### PR TITLE
Feature/search result detail view

### PR DESCRIPTION
**Pull request by alvare52**
# Description

Fixes Detail Screen works for search results
Trello card: https://trello.com/c/jWvqMtC6/137-book-details

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Change Status
- [x] Complete, tested, ready to review and merge

## How Has This Been Tested?
- [x] Manually tested with simulator (iPhone 11 Pro Max)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
awesome.

File: BetterReads/BetterReads/View Controllers/BookDetailViewController.swift [hunk 1]
```diff
@@ -185,6 +192,8 @@ class BookDetailViewController: UIViewController {
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
         navigationController?.navigationBar.barStyle = .default
+        navigationController?.navigationBar.tintColor = .black
```
Comment (ladybeitel) [inline BetterReads/BetterReads/View Controllers/BookDetailViewController.swift:195]: What color is the tint on the nav bar? Is it black or white? 
Either delete line 179 or 195 please.

**Review by ladybeitel — CHANGES_REQUESTED** [review]
Please review comments on Book Detail VC

**Review by ladybeitel — APPROVED** [review]

**Comment by ladybeitel** [comment]
[Book Details](https://trello.com/c/jWvqMtC6/137-book-details)

**Pull request merged**